code/modules/food_and_drinks/kitchen_machinery/gibber.dm

```diff
@@ -61,6 +61,7 @@
 
 /obj/machinery/gibber/RefreshParts()
 	var/gib_time = 40
+	meat_produced = 0
```

Comment:
This is doing to meat_produced what gib_time does for gibtime. You may want to handle both those vars in the same way just to keep it clean looking.